.github/workflows/ci.yml

```diff
@@ -83,10 +83,12 @@ jobs:
           bundle exec parallel_rspec -n $PARALLEL_TESTS_CONCURRENCY -e './bin/parallel_tests'
       - name: Check for missing annotations
         run: bundle exec annotate
-      - name: Check for untracked changes in app and spec directories
+      - name: Check for missing doc changes
+        run: ./bin/update-docs && git diff
```

Comment:
I'm a bit worried about running the whole suite twice, this will double our CI cost

Reply:
well it's not the whole suite, just the requests specs, but still it can be a lot of extra CI time 🤔 maybe we need to discuss what are our options, what a good tradeoff would look like

Reply:
also having to run all the specs locally to regenerate the docs could be a lot for big projects 🤔 is it possible to re-run only the changed specs or something like that?